Sources/MapboxCoreNavigation/RerouteControllerDelegate.swift

```diff
@@ -4,7 +4,10 @@ import MapboxNavigationNative
 
 
 protocol ReroutingControllerDelegate: AnyObject {
-    // TODO: fill with Native RerouteController and Alternative routes integration
+    func rerouteControllerWantsSwitchToAlternative(_ rerouteController: RerouteController,
```

Comment:
Maybe just `rerouteControllerDidSwitchToAlternative`?

Reply:
`did` does not reflect the case correctly, because when this delegate method is called, nothing is switched yet. Instead, it tells that Navigator has detected user switching to the known alternative route and suggests SDK to switch to it too. Technically we could ignore this call and route will not be switched, leaving user in "offroute" state.